01-ActivityModel/activity-model/config/lookups.yaml

```diff
@@ -0,0 +1,235 @@
+################################################################################
+# In this file is listed all lookups used by the Activity Model to organize the
+# EPC data to be merged with the Synthetic Population.
+#
+################################################################################
+# EPC data info:
+# - EPCs issued from January 2008 up to and including 30 June 2021.
+# - 21,440,172 Domestic EPCs
+# - Download: 18 October 2021
+# 
+# * Note that new EPCs may include keys not listed here!
+#
+################################################################################
+# EPC Headers:
+# Each file has 90 headers (keys). In this work, the used headers are:
+# - POSTCODE
+# - LOCAL_AUTHORITY
+# - PROPERTY_TYPE + BUILT_FORM = Accommodation type
+# - CONSTRUCTION_AGE_BAND
+# - TENURE    
+# - TOTAL_FLOOR_AREA  # m²
+# - NUMBER_HABITABLE_ROOMS
+#
+################################################################################
+# Dictionary lookups:
+# keys: based on the values present in the EPC
+# keys_values: based on the values present in the Synthetic Population
+#
+################################################################################
+# List lookups (accommodation age and floor area)
+# values: arbitrary chunks
+# 
+# List layout:
+# -
+#   - chunk desired code
+#   - chunk minimum value (not included)
+#   - chunk maximum value (included)
+################################################################################
+
+## Area lookup from: https://geoportal.statistics.gov.uk/
+area_url: "https://www.arcgis.com/sharing/rest/content/items/8a824519215947da99146692b0a0ff49/data"
+
+area_in_out:
+  - "pcds"
```

Comment:
"pcds" # What does this represent?

Reply:
I tried to run the model for MSOA areas, should I remove the "oa11cd" and replace for the code of the msoa describes in the readme?